### DISCUSSION TITLE
Regulatory approval

### DISCUSSION THREAD

**Discussion by peterantypas** [peterantypas/maiana#32] [Ideas]
Starting this topic to exchange ideas about regulatory status in different jurisdictions.

Here in the US, putting a device like this through the regulatory process would be gruelling: It will need a dedicated DSC receiver (yuck), FCC Part 80 testing and additional USCG approval. I'm not doing any of this, because I also have the right to build up to five transmitters without asking the FCC for any permission (thanks, ARRL!). 

So what I will do with MAIANA&trade; here in the US is _enable technically savvy sailors to build their own AIS transmitter_.

What do you folks know about Europe? I can't find any information about CE requirements for such devices.

**Comment by astuder** [comment]
I think for CE it would have to comply with all the IEC standards listed here:
http://www.allaboutais.com/index.php/en/standards/standards-of-ais

**Comment by astuder** [comment]
Test reports from commercial AIS transponders may be a good starting point to get an idea of the scope. For example:
https://www.oroliamaritime.com/wp-content/uploads/18-110247-EU-Type-Examination-Certificate-RED-Module-B.pdf
https://fccid.io/YYG-411-0002

**Reply by peterantypas** [reply]
Thanks, that's a good idea. This might be the right way to find a qualified lab as well, as neither Intertek nor Cetecom have any clue about CE RED for AIS ...

**Reply by astuder** [reply]
There's also this, which indicates that USCG approval is transferable to EU and vice versa.
https://mariners.coastguard.blog/2019/11/05/nvic-guide-to-marine-equipment-approvals-covered-by-a-mutual-recognition-agreement/

**Reply by peterantypas** [reply]
Interesting ... the document specifically mentions AIS too: 
![Screenshot from 2021-01-02 12-20-11](https://user-images.githubusercontent.com/1565933/103465911-e7d29180-4cf4-11eb-97fe-b9dc86b8b1a2.png)

The tougher problem is DSC though. I can't think of a low-cost solution for it.

**Reply by astuder** [reply]
And of course it also doesn't answer the question of the legality of using a DIY device even if it complies with all the standards.

**Reply by peterantypas** [reply]
Uber didn't become Uber by acquiring taxi licenses ...

**Reply by astuder** [reply]
But they've got armies of lawyers paid for by an infinite amount of OPM ;)

**Reply by rszemeti** [reply]
In the EU, you basically need a certificate, signed by an EU resident responsible for placing the device on the market, certifying conformity with the relevant directives.  No test house can issue this.  What a test house can do is issue the signatory with a report that gives them evidence that they have taken the required steps to ensure their device conforms. BUT, test house report or not, the responsibility comes back to the person who signs the document (usually the manufacturer or distributor) as if it's not compliant, you can be fined €5,000 *per device* ...  no signed certificate means no CE mark.  You don't apply for a CE mark, you just sign the certificate and put it on yourself .. but you better be sure you really comply!

Some equipment MUST be tested by a "notified body" ... ie a specially approved test house. I was not able to discover whether AIS devices needed to be tested by a notified body or not.

**Reply by peterantypas** [reply]
Aaaah, now I get it. Thanks for breaking it down. This puts a higher burden on the seller and requires an EU based distributor, plus a significant amount liability insurance, as one can never be 100% certain they're in compliance. No wonder prices are so high ...

**Reply by marcelrv** [reply]
That's why as a private person you can buy the cheaper devices from China you yourself import to EU, they even have CE, but you won't find any formal dealer/importer of that sort of equipment.

I also understood one of the major gripes of the lesser quality equipment is that they are less stable wrt the timing of the timeslot, so may not always comply fully to the standard.

**Reply by rszemeti** [reply]
> 
> 
> Aaaah, now I get it. Thanks for breaking it down. This puts a higher burden on the seller and requires an EU based distributor, plus a significant amount liability insurance, as one can never be 100% certain they're in compliance. No wonder prices are so high ...

Well, don't forget we also have sales tax (VAT, MwSt, TVA or whatever) of  20% which doesn't help ...

**Comment by rszemeti** [comment]
The MAY have an illegally applied CE mark, but they 100% absolutely do NOT
have a declaration of conformity so they do not have CE.  There is a very
similar "China Export" mark that is coincidentally almost identical to the
CE mark ...

**Comment by peterantypas** [comment]
Thanks everyone, this has been very enlightening. Now, for the 64K question:

Have any of you heard of any authorities in Europe switching AIS traffic away from the default frequencies? Here in the US the Coast Guard did it by mistake once in 2010 in the East Coast.

**Comment by rszemeti** [comment]
I know the UK licence specifies the normal three frequencies for AIS as:
The channels allocated for AIS are:AIS 1 (161.975MHz), AIS 2(162.025MHz),and
VHF channel 70 (156.525MHz).
So in theory at least, no normal stations are even licensed to use anything
other than those frequencies.

**Reply by peterantypas** [reply]
For class B devices, the only use of DSC on channel 70 is for frequency management. According to the standard, a class B device should listen for AIS traffic first to establish which frequencies are used, then monitor DSC channel 70 periodically. I posit that this is an unnecessary burden as the odds of a frequency change in practice are almost zero. I suspect a lot of these Chinese devices out there don't even have any such mechanisms.

**Comment by rszemeti** [comment]
I would not be surprised if even some of the approved equipment doesn't
bother ...

**Reply by peterantypas** [reply]
Speaking of approved equipment, I can positively attest to the fact that there are commercial class A stations out there that literally walk all over the SOTDMA timeslots, so they fail even at the most fundamental level.
